File: java/picocontainer/src/test/org/picocontainer/defaults/DefaultLazyInstantiationTestCase.java
package org.picocontainer.defaults;

import org.picocontainer.tck.AbstractLazyInstantiationTestCase;
import org.picocontainer.MutablePicoContainer;

/**
 * @author Aslak Helles&oslash;y
 * @version $Revision$
 */
public class DefaultLazyInstantiationTestCase extends AbstractLazyInstantiationTestCase {
    protected MutablePicoContainer createPicoContainer() {
        return new DefaultPicoContainer();
    }
}

File: dotnet/picocontainer/src/Test/Defaults/DefaultLazyInstantiationTestCase.cs
/*****************************************************************************
 * Copyright (C) PicoContainer Organization. All rights reserved.            *
 * ------------------------------------------------------------------------- *
 * The software in this package is published under the terms of the BSD      *
 * style license a copy of which has been included with this distribution in *
 * the license.txt file.                                                     *
 *                                                                           *
 * Idea by Rachel Davies, Original code by Aslak Hellesoy and Paul Hammant   *
 * C# port by Maarten Grootendorst                                           *
 *****************************************************************************/

using System;
using System.Diagnostics;

using csUnit;

using PicoContainer.Defaults;
using PicoContainer.Tests.Tck;
using PicoContainer.Tests.TestModel;

namespace PicoContainer.Tests.Defaults 
{

	/// Summary description for DefaultLazyInstantiationTestCase.
	/// </summary>
	public class DefaultLazyInstantiationTestCase :AbstractLazyInstantiationTestCase
	{
    protected override MutablePicoContainer createPicoContainer() 
    {
      return new DefaultPicoContainer();
    }
  }
}
